aws-yaml-ci.yml

```diff
@@ -0,0 +1,177 @@
+# Variable 'AWS_ACCESS_KEY_ID' defined in the Variable Group tab
+# Variable 'AWS_SECRET_ACCESS_KEY' defined in the Variable Group tab
+# Variable 'awsCredentials' defined in the Variable Group tab
+# Variable 'CookbooksSource' defined in the Variables tab
+# Variable 'CookbooksSource1' defined in the Variables tab
+# Variable Group 'Default Branch' defined in the Variables tab
+# Variable Group 'Template Parameters' defined in the Variables tab
+# Variable Group 'VPC ap-southeast-2' defined in the Variables tab
+# Variable Group 'ImageVersion' defined in the Variables tab
+variables:
+- name: captureStackOutputs
+  value: asJSON
+- name: CookbooksSource
+  value: $(Build.Repository.LocalPath)\cookbooks
+- name: CookbooksSource1
+  value: c:\lansa
+- name: regionName
+  value: ap-southeast-2
+- group: "Default Branch"
+- group: "Template Parameters"
+- group: "VPC ap-southeast-2"
+- group: "ImageVersion"
+- group: "ImageVersion - New"
```

Comment:
Why has this new variable group been created? Put the new 2022 variables in the ImageVersion variable group. Delete "ImageVersion - New"

Reply:
Variable group 'ImageVersion - New' deleted and windows 2022 values added in 'ImageVersion' group.